recipes/hoggormplot/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hoggormplot" %}
+{% set version = "0.13.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hoggormplot-{{ version }}.tar.gz
+  sha256: cc34395e11a8b66b62c3eedd37dd9b2ce987f75b4e041122728ebb7d8a95bb51
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - sphinx
```

Comment:
It doesn't look like `sphinx` is actually used to build the package, so you can remove that. Also, you don't need `hoggorm` and `matplotlib-base` under 'host', just leave them under `run`.

Reply:
The original package says "setup_requires = Sphinx", so it's needed at the moment for building the package.
But do you have a link where I can best familiarize myself with the topic because then I could make a change request to the original project? Also hoggorm is needed at the moment (other the package fails for linux.)

Reply:
Yeah, I missed the `version = attr: hoggormplot.__version__` in the `setup.cfg`, which causes the package to be imported when running `setup.py`, so anything needed at import time will need to be included on `host`.